File: Modul5/Python/PRAK502-2310817210003-MuhammadAzrianzan.py
def mutlak(nilai):
    if nilai < 0:
        nilaiMutlak = -(nilai)
        return nilaiMutlak
    else:
        return nilai

def hitung(nilai1, nilai2):
    hasilHitung = nilai1 - nilai2
    return mutlak(hasilHitung)

a, c, b, d = input("").split()
a = int(a)
b = int(b)
c = int(c)
d = int(d)
Hasil = hitung(a,b) + hitung(c,d)
print(mutlak(Hasil))

File: Modul5/C/PRAK502-2310817210003-MuhammadAzrianzan.c
#include <stdio.h>
#include <math.h>

int mutlak(int nilai){
    if (nilai < 0) {
        int nilaiMutlak = -(nilai);
        return nilaiMutlak;
    }
    else {
        return nilai;
    }
}

int hitung(int nilai1, int nilai2){
    int hasilHitung = nilai1 - nilai2;
    return mutlak(hasilHitung);
}

int main() {
    int a,b,c,d, Hasil;
    scanf("%d %d %d %d", &a, &c, &b, &d);
    Hasil = hitung(a,b) + hitung(c,d);
    printf("%d",mutlak(Hasil));
    return 0;
}